code/system/application/perception/include/detectlane/detectlane.hpp

```diff
@@ -56,34 +58,50 @@ class DetectLane
  private:
   void setUp();
   void tearDown();
-  odcore::data::dmcp::ModuleExitCodeMessage::ModuleExitCode body();
+  // odcore::data::dmcp::ModuleExitCodeMessage::ModuleExitCode body();
```

Comment:
You should remove this line since it not relevant in a DataTriggered (the commented line may confuse a new developer).